core/server/data/migration/populate.js

```diff
@@ -31,9 +31,9 @@ populate = function populate(logger, tablesOnly) {
 
     return tableSequence.then(function () {
         // Load the fixtures
-        return fixtures.populate(logger);
+        return fixtures.populate(logger, modelOptions);
     }).then(function () {
-        return fixtures.ensureDefaultSettings(logger);
+        return models.Settings.populateDefaults(modelOptions);
```

Comment:
i deleted populating settings default for populate or update within the migrations module completly in #7019